File: tks/RestAdapters/src/main/kotlin/dto/UserDto.java
package dto;

import lombok.Data;

import javax.json.bind.annotation.JsonbTransient;

@Data
public class UserDto {
    private String email;
    private String password;
    private boolean active;
    private String role;

    @JsonbTransient
    public String getPassword() {
        return password;
    }
}

File: tks/Microservices/user-service/src/main/kotlin/edu/p/lodz/pl/service/users/dto/UserDto.kt
package edu.p.lodz.pl.service.users.dto

import javax.json.bind.annotation.JsonbTransient

data class UserDto(
    var email: String? = null,
    @get:JsonbTransient var password: String? = null,
    var active: Boolean? = null,
    var role: String? = null
)